src/meta/play-meta.js

```diff
@@ -426,5 +427,18 @@ export const plays = [
     blog: "",
     video: "",
     language: "js",
+  }, {
+    id: 'pl-lorem-generator', 
+    name: 'Lorem Generator', 
+    description: 'A basic Lorem Ipsum generator with copy functionality. Made with TailwindCSS and ReactJS',
+    component: () => {return <LoremGenerator />},
+    path: '/plays/lorem-generator',
+    level: 'Intermediate',
+    tags: 'Hooks,TailwindCSS,JSX,API',
+    github: 'madflows',
+    cover: '',
```

Comment:
Can you please drop a cover.png file inside your play folder for the thumbnail to appear?

Reply:
We need a cover.png file to show the thumbnail image of your play. Kindly drop one in your play folder and push. Please let me know if you need any help.

![image](https://user-images.githubusercontent.com/3633137/174936139-2b51e093-6df5-4849-8aee-d0cf7b4a9365.png)